pkg/scheduler/framework/session.go

```diff
@@ -128,18 +128,28 @@ func closeSession(ssn *Session) {
 			continue
 		}
 
+		// patch the backfilled annotation
+		for _, task := range job.Tasks {
+			annotation := make(map[string]string)
```

Comment:
How about creating the map after the `if` statement on line 135?

Reply:
fixed